.github/workflows/lint.yml

```diff
@@ -105,6 +105,7 @@ jobs:
                      --known-failure app/reporting/reporting.cpp \
                      --known-failure app/reporting/tests/MockReportScheduler.cpp \
                      --known-failure app/reporting/tests/MockReportScheduler.h \
+                     --known-failure app/server/TermsAndConditionsManager.cpp \
```

Comment:
Why is this a known failure?

Reply:
Updated code layout to create a separate source set that also includes the cpp file.